utils/appverifier_ctest_util.py

```diff
@@ -0,0 +1,84 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+# Built-in
+from asyncio import subprocess
+import argparse
+import subprocess
+import os
+import json
+# Needs to be installed via pip
+from termcolor import colored  # - for terminal colored output
```

Comment:
I personally prefer not installing dependencies over getting red output

just prefix errors with something easily searchable, like "ERROR:"